.github/workflows/ci-build.yml

```diff
@@ -57,6 +57,53 @@ jobs:
           echo $'Examples\n========' > doc/source/examples/index.rst
           make -C doc html SPHINXOPTS="-j auto -w build_errors.txt -N"
 
+      - name: Build json documentation
+        run: |
+          # Export the documentation files to JSON files.
+          sphinx-build -M json doc/source doc-json -j auto -w build_errors.txt -N;
+
+      - name: Flatten the generated nested files into a single directory
+        run: |
+          echo Flattening a nested directory
+          mkdir doc-flatten-json;
+          echo Move all the JSON file to the flatten directory.
+          find . -name "*.fjson" -exec mv "{}" --backup=numbered ./doc-flatten-json \;
+          echo Make sure all the file has .json extensions instead of the .fjson
+          for f in doc-flatten-json/* ; do mv -- "$f" "$f.json" ; done;
+          echo Move all static image files to the JSON flatten folder.
+          mv doc-json/json/_images doc-flatten-json/_images;
```

Comment:
Also the images are in a folder:

![image](https://user-images.githubusercontent.com/28149841/193007201-6afb22eb-5430-43ef-9fc8-7f01682309f5.png)

Not sure if that is the idea.

Reply:
For the first draft, it should be OK from the feedback I got.
But I completely agree that this could evolve in the next set of modifications where I will be handling examples for the "Full" documentation build for each release.